plugin/src/main/java/com/redhat/jenkins/plugins/ci/authentication/activemq/UsernameAuthenticationMethod.java

```diff
@@ -87,6 +88,7 @@ public ActiveMQConnectionFactory getConnectionFactory(String broker) {
     }
 
     @Extension
+    @Symbol("password")
```

Comment:
I would maybe use "**basicAuth**" instead of password.

Reply:
Yeah, I do not like the proposed name very much, too. Though, is it using HTTP Basic Auth under the hoods?

Reply:
I guess `basic`, `simple` and `password` would work equally well for me. The most descriptive `usernamePassword` is annoyingly repetitive given it is followed by `username` and `password` elements.

@scoheb, any preference?

Reply:
`simple` seems to fit nice